http.go

```diff
@@ -280,18 +281,25 @@ func (c *httpFS) send(httpClient *http.Client, req *http.Request) (*http.Respons
 // access fs. It should be accessed by clients created using this
 // package's HTTP func.
 func HTTPHandler(fs FileSystem, logTo io.Writer) http.Handler {
+	return HTTPHandlerWithDelay(fs, logTo, 0)
+}
+
+func HTTPHandlerWithDelay(fs FileSystem, logTo io.Writer, delay time.Duration) http.Handler {
 	if logTo == nil {
 		logTo = ioutil.Discard
 	}
-	return &httpFSHandler{fs, log.New(logTo, "rwvfs HTTP: ", log.Flags())}
+	return &httpFSHandler{fs, log.New(logTo, "rwvfs HTTP: ", log.Flags()), delay}
 }
 
 type httpFSHandler struct {
-	fs  FileSystem
-	log *log.Logger
+	fs    FileSystem
+	log   *log.Logger
+	delay time.Duration
 }
 
 func (h *httpFSHandler) ServeHTTPAndReturnError(w http.ResponseWriter, r *http.Request) error {
+	time.Sleep(h.delay)
```

Comment:
How big of a deal would it be that when when delay is not set, you'll be calling `time.Sleep(0)` for every `ServeHTTPAndReturnError` call?

Probably really really not a big deal at all. But just in case you think it's worth avoiding it, it'd be very easy to rewrite this as a wrapper adapter on top of the existing type (or, more generally, over `http.Handler` interface):

```Go
type handlerWithDelay struct {
	http.Handler
	delay time.Duration
}

func (h *handlerWithDelay) ServeHTTP(w http.ResponseWriter, req *http.Request) {
	time.Sleep(h.delay)
	h.Handler.ServeHTTP(w, req)
}
```